packages/material-ui/src/Tooltip/Tooltip.test.js

```diff
@@ -84,88 +77,74 @@ describe('<Tooltip />', () => {
-    const children = wrapper.childAt(0).childAt(0);
+    const children = wrapper.find('#testChild');
     assert.strictEqual(handleRequestOpen.callCount, 0);
     assert.strictEqual(handleClose.callCount, 0);
-    children.simulate('mouseOver', { type: 'mouseover' });
+    children.simulate('mouseOver');
     assert.strictEqual(handleRequestOpen.callCount, 1);
     assert.strictEqual(handleClose.callCount, 0);
-    children.simulate('mouseLeave', { type: 'mouseleave' });
+    children.simulate('mouseLeave');
     assert.strictEqual(handleRequestOpen.callCount, 1);
     assert.strictEqual(handleClose.callCount, 1);
   });
 
   it('should close when the interaction is over', () => {
-    const wrapper = shallow(<Tooltip {...defaultProps} />);
-    const childrenRef = document.createElement('div');
-    childrenRef.tabIndex = 0;
-    wrapper.instance().childrenRef = childrenRef;
-    const children = wrapper.childAt(0).childAt(0);
-    assert.strictEqual(wrapper.instance().state.open, false);
-    children.simulate('mouseOver', { type: 'mouseover' });
-    childrenRef.focus();
-    children.simulate('focus', { type: 'focus', persist });
-    clock.tick(0);
-    assert.strictEqual(wrapper.instance().state.open, true);
-    children.simulate('mouseLeave', { type: 'mouseleave' });
-    assert.strictEqual(wrapper.instance().state.open, false);
-    children.simulate('blur', { type: 'blur' });
-    assert.strictEqual(wrapper.instance().state.open, false);
+    const wrapper = mount(<Tooltip {...defaultProps} />);
+    const children = wrapper.find('#testChild');
+    assert.strictEqual(wrapper.find(Popper).props().open, false);
+    children.simulate('mouseOver');
+    children.simulate('focus');
+    assert.strictEqual(wrapper.find(Popper).props().open, true);
+    children.simulate('mouseLeave');
+    assert.strictEqual(wrapper.find(Popper).props().open, false);
+    children.simulate('blur');
+    assert.strictEqual(wrapper.find(Popper).props().open, false);
   });
 
   describe('touch screen', () => {
     it('should not respond to quick events', () => {
-      const wrapper = shallow(<Tooltip {...defaultProps} />);
-      const childrenRef = document.createElement('div');
-      childrenRef.tabIndex = 0;
-      wrapper.instance().childrenRef = childrenRef;
-      const children = wrapper.childAt(0).childAt(0);
-      children.simulate('touchStart', { type: 'touchstart', persist });
-      children.simulate('touchEnd', { type: 'touchend', persist });
-      childrenRef.focus();
-      children.simulate('focus', { type: 'focus', persist });
-      assert.strictEqual(wrapper.instance().state.open, false);
+      const wrapper = mount(<Tooltip {...defaultProps} />);
+      const children = wrapper.find('#testChild');
+      children.simulate('touchStart');
+      children.simulate('touchEnd');
+      children.simulate('focus');
+      assert.strictEqual(wrapper.find(Popper).props().open, false);
```

Comment:
I can still find the tooltip in the dom but the Popper is closed

Reply:
This is probably due to this happening in the next render cycle. `react-transition-group` always is one render phase behind which is probably why we need to change the transitions in the Modal #15266 and Popper #15250.

I guess checking the Popper prop is fine then.